.github/workflows/e2e-ci-playwright.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          ref: '${{ github.event.pull_request.merge_commit_sha }}'
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
```

Comment:
@kelvah @t-hendricks Hi, I came across an issue where the Playwright CI was not consistently running against the latest commit. I have addressed this and pushed an update today.
Could you please take a moment to re-review the changes when you get a chance? Thanks a lot